chart/values.yaml

```
@@ -288,6 +288,7 @@ workers:
 
 # Airflow scheduler settings
 scheduler:
+  replicas: 2
```

```suggestion
  replicas: 1
```

Default needs to stay at 1 I think